tests/typecheck/models/test_contrib_models.yml

```diff
@@ -1,3 +1,16 @@
+-   case: contrib_admin_model_fields
```

Comment:
These tests are redundant with the `assert_type` matching one ?
Do we need them still ?

Reply:
ok so from what I can see in workflow these tests are evaluated only using mypy via pytest (the ones under tests/typecheck/models/test_contrib_models.yml). The ones under `assert_type/` are evaluated using pyright https://github.com/typeddjango/django-stubs/blob/3e395fa5af4bd64e1aa0a3086c029fb57b3baed5/.github/workflows/test.yml#L123
```json
    "include": [
        "tests/assert_type/"
    ],
```
Ideally we would want pyright and mypy to produce same results for all model fields, so I think having them as separate tests help in case behavior of one of the type checkers change is the future or any possible changes in the stubs too which might affect this.

Reply:
fair enough, I didn't really take them out since one benefit of having them on pytest side is few people might already be familiar running them with just `pytest` but as opposed to running a few ci commands locally. But this might be more suited for a change in another pr along with a contributor guidlines update